docs/relax.md

```diff
@@ -58,6 +67,75 @@ Running `reduce` will remove the redundant axiom (5), leaving the following axio
 
 This SubClassOf graph is complete and non-redundant, and can be used for intuitive visualization and browsing of the ontology
 
+## Note about normalization of Qualified Number restrictions
+
+For convenience of downstream processing, `relax` rewrites expressions of the kind `:R min 1 :A` to `:R some :A`. These two expressions are logically identical.
```

Comment:
What about `:R min 2 :A` (or any number greater than zero)?

Reply:
```
else if (x instanceof OWLObjectCardinalityRestriction) {
      OWLObjectCardinalityRestriction ocr = (OWLObjectCardinalityRestriction) x;
      OWLClassExpression filler = ocr.getFiller();
      OWLObjectPropertyExpression p = ocr.getProperty();
      if (ocr.getCardinality() > 0) {
        OWLObjectSomeValuesFrom svf = dataFactory.getOWLObjectSomeValuesFrom(p, filler);
        svfs.add(svf);
      }
```

All of them are rewritten actually. 

```suggestion
For convenience of downstream processing, `relax` rewrites expressions of the kind `:R min 1 :A` or `:R min 2 :A` to `:R some :A`. This is safe, because `:R some :A` is implied by any cardinality restriction > 0.
```

Reply:
Thanks for pointing that out, clarified the docs.